src/components/DatasetItem.vue

```diff
@@ -209,14 +238,33 @@
 
      disabledClass() {
        return this.disabled ? "ds-item-disabled" : "";
+     },
+
+     formatFdr() {
+       return JSON.parse(this.dataset.fdrCounts)[1];
      }
    },
    data() {
      return {
        showMetadataDialog: false,
+       opticalImageSmall: null,
        disabled: false
      };
    },
+
+   mounted() {
+     this.$apollo.query({
```

Comment:
I'd propose to move this query to an `apollo` property. You can find an example in `ImageAlignmentPage.vue` for instance. It organizes the code better, and will certainly be more handy if we want to add more GQL queries in this component in the future